### ISSUE TITLE
Autorotate Elasticsearch

### ISSUE THREAD

**Issue by tomjirsa** [CSIRT-MU/Stream4Flow#20]
Explore possibilities for data rotation in elasticsearch (i.e. delete all indexes older than one month). 

- first, look for native settings in elasticsearch (whether the autorotate is possible)
- second, look for other solutions (automatic scripts using e.g. cron)
- present and chose solution
- implement solution to ansible - if possible, make the rotation size parametrizable in global vars.

**Comment by simkosvorad** [comment]
Elasticsearch curator is i think the thing we are looking for [url](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/about.html) exactly [url](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/ex_delete_indices.html) action. Also can be ran as a cron job.

**Comment by tomjirsa** [comment]
Try to look at [Elastic Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/about.html) on [github](https://github.com/elastic/curator). 

Suggested in [StackOverflow](http://stackoverflow.com/questions/29966977/how-to-control-and-rotate-storage-data-at-elasticsearch) and [Elastic discuss forum](https://discuss.elastic.co/t/index-rotation-for-logstash-users/25130/3)

**Comment by simkosvorad** [comment]
To test autorate you can use
```
curl -XPUT 'consumer:9200/spark-2017.03.01?pretty' -H 'Content-Type: application/json' -d'
{
    "settings" : {
        "index" : {
            "number_of_shards" : 3,
            "number_of_replicas" : 2
        }
    }
}
'
```
for creating an index.
to list all indices you can use:
`curl 'consumer:9200/_cat/indices?v'
`

**Comment by tomjirsa** [comment]
Closing, merged with master #31
